code/datums/traits/negative.dm

```diff
@@ -100,94 +100,337 @@
+		if("Scientist")
+			sci = TRUE
+			heirloom_table.Add(/obj/item/nanite_remote)
+		if("Exploration Crew")
+			sci = TRUE
+			heirloom_table.Add(/obj/item/throwing_star/toy)
+		if("Roboticist")
+			sci = TRUE
+			heirloom_table.Add(pick(subtypesof(/obj/item/toy/prize)), //look at this nerd
+						   	   /obj/item/book/manual/wiki/medicine)
+		//Medical
+		if("Chief Medical Officer")
+			comm = TRUE
+			med = TRUE
+			heirloom_table.Add(/obj/item/book/manual/wiki/chemistry,
+						   	   /obj/item/book/manual/wiki/infections,
+						   	   /obj/item/reagent_containers/dropper,
+						   	   /obj/item/healthanalyzer)
+		if("Brig Physician")
+			med = TRUE
+			sec = TRUE
+			heirloom_table.Add(/obj/item/healthanalyzer)
+		if("Medical Doctor")
+			med = TRUE
+			heirloom_table.Add(/obj/item/healthanalyzer)
+		if("Paramedic")
+			med = TRUE
+			heirloom_table.Add(/obj/item/healthanalyzer)
+		if("Psychiatrist")
+			med = TRUE
+			heirloom_table.Add(/obj/item/healthanalyzer)
+		if("Chemist")
+			med = TRUE
+			heirloom_table.Add(/obj/item/book/manual/wiki/chemistry,
+						   	   /obj/item/storage/bag/chemistry,
+						   	   /obj/item/reagent_containers/dropper)
+		if("Virologist")
+			med = TRUE
+			heirloom_table.Add(/obj/item/book/manual/wiki/infections,
+						   	   /obj/item/book/manual/wiki/chemistry,
+						   	   /obj/item/storage/bag/bio,
+						   	   /obj/item/reagent_containers/food/drinks/bottle/virusfood,
+						   	   /obj/item/reagent_containers/dropper)
+		if("Geneticist")
+			sci = TRUE
+			med = TRUE
+			heirloom_table.Add(/obj/item/nanite_remote)
+		//Engineering
+		if("Chief Engineer")
+			comm = TRUE
+			engi = TRUE
+			heirloom_table.Add(/obj/item/clothing/head/hardhat/white)
+		if("Station Engineer")
+			engi = TRUE
+			heirloom_table.Add(/obj/item/clothing/head/hardhat)
+		if("Atmospheric Technician")
+			engi = TRUE
+			heirloom_table.Add(/obj/item/lighter,
+						   	   /obj/item/lighter/greyscale,
+						   	   /obj/item/storage/box/matches,
+						   	   /obj/item/tank/internals/emergency_oxygen/empty)
+		//Supply
+		if("Quartermaster")
+			supp = TRUE
+			heirloom_table.Add(/obj/item/stamp,
+						   	   /obj/item/stamp/denied)
+		if("Cargo Technician")
+			supp = TRUE
+			heirloom_table.Add(/obj/item/clipboard)
+		if("Shaft Miner")
+			supp = TRUE
+			heirloom_table.Add(/obj/item/pickaxe/mini,
+						   	   /obj/item/shovel)
+		//Other
+		if("Captain")
+			comm = TRUE
+			heirloom_table.Add(/obj/item/reagent_containers/food/drinks/flask/gold,
+						   	   /obj/item/reagent_containers/food/drinks/flask/gold,
+						   	   /obj/item/reagent_containers/food/drinks/flask/gold)
+			//hich chance of spawning captain's flask
+		if("Head of Personnel")
+			comm = TRUE
+			civ = TRUE
+			heirloom_table.Add(/obj/item/toy/plush/ian)
+		//---End of Switch If lines---
+
+	// 3.Department specific table
+	// Note: do not use 'else if' here because certain jobs are in multiple departments.
+	if(civ || prob(1))
```

Comment:
See what i said for species. I'd say dont even make this its own switch(), just include it in the job switch()

Reply:
fixed. thanks.